src/compiler/moduleSpecifiers.ts

```diff
@@ -1,484 +1,422 @@
-// Used by importFixes, getEditsForFileRename, and declaration emit to synthesize import module specifiers.

```

Comment:
@weswigham looks like this comment got lost during "terraforming"

Reply:
Yeah, looks like I should copy the comment ranges from the namespace declaration I remove onto the first declaration inside the namespace.